go.mod

```diff
@@ -151,3 +151,5 @@ require (
 	modernc.org/strutil v1.2.0 // indirect
 	modernc.org/token v1.1.0 // indirect
 )
+
+replace go.temporal.io/api => github.com/temporalio/api-go v1.34.1-0.20240528230324-f948449f09fc
```

Comment:
fetching the `api-go` of my feature: https://github.com/temporalio/api-go/tree/shivam-backlog-stats

Reply:
sound good. I would keep this comment here as a reminder until you do that